security/tls-authorization/README.adoc

```diff
@@ -0,0 +1,226 @@
+= Remote caches with TLS (SSL) encryption and security authorization
+
+**Authors:** Wolf Fink +
+**Technologies:** Infinispan, Hot Rod, Java, SSL/TLS, RBAC +
+**Summary:** Use SSL/TLS certificates to verify Hot Rod client identities and encrypt network traffic to the server. This tutorial also demonstrates how to perform client certificate authentication and use security authorization with client certificates.
+
+== Building instructions and preparation
+
+. `$ mvn clean package`
+. Build the example certificates and keystores with the following scripts or directly with the Java keytool commands:
+* `create_unsigned_certificates.sh`
+* `create_signed_certificates.sh`
++
+These scripts create keystores and trust stores in the project directory.
+
+. Make sure you are using Java 11 at a minimum.
++
+Infinispan Server requires at least Java 11.
+Additionally the example keystores and trust stores in this tutorial use `PKCS12`, which is the recommended standard and the default from Java 11.
+. Download and start at least one Infinispan Server instance.
+
+== Unsigned Server certificates
+
```

Comment:
First step should be to "Run `create_unsigned_certificates.sh` to generate a server keystore and client trust store with unsigned certificates."